packages/angular_devkit/core/src/index.ts

```diff
@@ -6,6 +6,7 @@
  * found in the LICENSE file at https://angular.io/license
  */
 import * as experimental from './experimental';
+import * as jobs from './jobs';
```

Comment:
Can you export using `./jobs/index`?

This is required since once the API guardian Pr lands, since it uses classic module resolution instead of node. Which means that this API won’t be discovered.

Reply:
Done.